python/doc/theory/data_analysis/chi2_fitting_test.rst

```diff
@@ -1,62 +1,63 @@
+
+We want to
+test whether the sample is drawn from the discrete distribution characterized by the
+probabilities :math:`\left\{ p(x;\vect{\theta}) \right\}_{x \in \cE}` and
+which cumulative distribution function is denoted by :math:`G`.
+
+This test  involves the calculation of the test statistic which is
+the distance between the empirical number of values equal to :math:`x` in the sample and the
+theoretical mean one evaluated from the discrete distribution.
+
+Let :math:`X_1, \ldots , X_{\sampleSize}` be i.i.d. random variables following the
+distribution with CDF :math:`F`. According to the tested distribution :math:`G`,
+the theoretical mean number of values equal to :math:`x` is :math:`\sampleSize p(x;\vect{\theta})`
+whereas the number evaluated from the :math:`X_1, \ldots , X_{\sampleSize}` is
+:math:`N(x) = \sum_{i=1}^{\sampleSize} 1_{X_i=x}`.
+Then the test statistic is defined by:
 
   .. math::
 
-     \begin{aligned}
-         \widehat{D}_N^2 = \sum_{x \in \cE_N} \frac{\left(Np(x)-n(x)\right)^2}{n(x)}
-       \end{aligned}
-
-where :math:`\cE_N` denotes the elements of :math:`\cE` which have
-been observed at least once in the data sample and where :math:`n(x)`
-denotes the number of data values in the sample that are equal to
-:math:`x`.
-
-| The probability distribution of the distance :math:`\widehat{D}_N^2`
-  is asymptotically known (i.e. as the size of the sample tends to
-  infinity), and this asymptotic distribution does not depend on the
-  candidate distribution being tested. If :math:`N` is sufficiently
-  large, this means that for a probability :math:`\alpha`, one can
-  calculate the threshold / critical value) :math:`d_\alpha` such that:
-
--  if :math:`\widehat{D}_N>d_{\alpha}`, we reject the candidate
-   distribution with a risk of error :math:`\alpha`,
-
--  if :math:`\widehat{D}_N \leq d_{\alpha}`, the candidate distribution
-   is considered acceptable.
-
-An important notion is the so-called “:math:`p`-value” of the test. This
-quantity is equal to the limit error probability
-:math:`\alpha_\textrm{lim}` under which the candidate distribution is
-rejected. Thus, the candidate distribution will be accepted if and only
-if :math:`\alpha_\textrm{lim}` is greater than the value :math:`\alpha`
-desired by the user. Note that the higher
-:math:`\alpha_\textrm{lim} - \alpha`, the more robust the decision.
+         D_{\sampleSize} = \sum_{x \in \cE} \frac{\left[\sampleSize p(x)-N(x)\right]^2}{N(x)}
+
+If some values of :math:`x` have not been observed in the sample, we have to gather values in
+classes so that they contain at least 5 data points (empirical rule). Then the theoretical
+probabilities of all the values in the class are added to get the
+theoretical probability of the class.
+
+Let :math:`d_{\sampleSize}` be the realization of the test statistic :math:`d_{\sampleSize}`
+on the sample :math:`\left\{ x_1,\dots,x_{\sampleSize} \right\}`.
+
```

Comment:
This blank line seems unnecessary to me, since the next sentence continues the flow of the previous one.
```suggestion
```